editors/code/src/main.ts

```diff
@@ -269,8 +269,8 @@ function serverPath(config: Config): string | null {
 async function isNixOs(): Promise<boolean> {
     try {
         const contents = (await vscode.workspace.fs.readFile(vscode.Uri.file("/etc/os-release"))).toString();
-        const idString = contents.split('\n').find((a) => a.startsWith("ID="));
-        return idString?.toLowerCase()?.indexOf("nixos") !== -1;
+        const idString = contents.split('\n').find((a) => a.startsWith("ID=")) || "ID=linux";
+        return idString.indexOf("nixos") !== -1;
```

Comment:
Are you still sure you don't want this :smile:?

```suggestion
        return contents.indexOf("nixos") !== -1;
```

Reply:
I still kinda am — almost sure two-three years from now someone will find a way to trigger it accidentally :D